signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/view/SignInWithAppleButton.kt

```diff
@@ -34,19 +60,29 @@ class SignInWithAppleButton @JvmOverloads constructor(
         val attributes = context.theme.obtainStyledAttributes(attrs, R.styleable.SignInWithAppleButton, 0, 0)
 
         val colorStyleIndex =
-            attributes.getInt(R.styleable.SignInWithAppleButton_sign_in_with_apple_button_colorStyle, SignInColorStyle.BLACK.ordinal)
+            attributes.getInt(
+                R.styleable.SignInWithAppleButton_sign_in_with_apple_button_colorStyle,
+                SignInColorStyle.BLACK.ordinal
+            )
         val colorStyle = SignInColorStyle.values()[colorStyleIndex]
 
-        val text = attributes.getInt(R.styleable.SignInWithAppleButton_sign_in_with_apple_button_textType, SignInTextType.SIGN_IN.ordinal)
+        val text = attributes.getInt(
+            R.styleable.SignInWithAppleButton_sign_in_with_apple_button_textType,
+            SignInTextType.SIGN_IN.ordinal
+        )
 
-        val cornerRadius = attributes.getDimension(R.styleable.SignInWithAppleButton_sign_in_with_apple_button_cornerRadius, resources.getDimension(R.dimen.sign_in_with_apple_button_cornerRadius_default))
+        val cornerRadius = attributes.getDimension(
+            R.styleable.SignInWithAppleButton_sign_in_with_apple_button_cornerRadius,
+            resources.getDimension(R.dimen.sign_in_with_apple_button_cornerRadius_default)
+        )
```

Comment:
Editor got a little overzealous? Can I prevent this with lint settings somewhere?